DuckDuckGo/AppDependencyProvider.swift

```diff
@@ -152,7 +152,7 @@ class AppDependencyProvider: DependencyProvider {
         networkProtectionKeychainTokenStore = NetworkProtectionKeychainTokenStore(keychainType: .dataProtection(.unspecified),
                                                                                   serviceName: "\(Bundle.main.bundleIdentifier!).authToken",
                                                                                   errorEvents: .networkProtectionAppDebugEvents,
-                                                                                  isSubscriptionEnabled: accountManager.isUserAuthenticated,
+                                                                                  isSubscriptionEnabled: true,
```

Comment:
This was the core issue, this was being used for the lifecycle of the app, so if you launched the app when you were not signed in, we didn't go through the subscription flow. Since the subscription is always enabled (this means "has launched", not "is signed in"), let's just hardcode this to true until we can remove this code completely.